src/Commands/core/UpdateDBCommands.php

```diff
@@ -364,8 +364,9 @@ public static function updateDoOnePostUpdate($function, DrushBatchContext $conte
      */
     public static function updateFinished($success, $results, $operations)
     {
-        // No code needed but the batch result bookkeeping fails without a finished callback.
-        $noop = 1;
+        // Some (post)updates might install/uninstall extensions that require
+        // clearing the static and persistent caches and container invalidation.
```

Comment:
This comment is now outdated, it still refers to the original approach.

I would make sure this mentions that this is done to mirror the approach taken in core, and link to the code responsible (`DbUpdateController::batchFinished()`).

Example:

```
// Flush all caches at the end of the batch operation. When Drupal core performs database
// updates it also clears the cache at the end. This ensures that we are compatible with updates
// that rely on this behavior.
// @see \Drupal\system\Controller\DbUpdateController::batchFinished()
```